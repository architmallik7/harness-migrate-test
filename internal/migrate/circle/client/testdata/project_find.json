{
    "slug": "circleci/216caad37819ng3EFmqdLS/T216caad37819NpwJ368",
    "organization_name": "Jane Citizen",
    "organization_id": "71555d36-fdfe-464d-95be-b0998480d9fb",
    "name": "hello-world",
    "id": "b9b97877-5d65-4a25-9a77-5e841f086b56",
    "organization_slug": "circleci/216caad37819ng3EFmqdLS",
    "vcs_info": {
        "vcs_url": "//circleci.com/71555d36-fdfe-464d-95be-b0998480d9fb/b9b97877-5d65-4a25-9a77-5e841f086b56",
        "default_branch": "main",
        "provider": "CircleCI"
    }
}