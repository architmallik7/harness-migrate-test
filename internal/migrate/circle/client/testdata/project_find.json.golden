{
  "id": "b9b97877-5d65-4a25-9a77-5e841f086b56",
  "name": "hello-world",
  "slug": "circleci/216caad37819ng3EFmqdLS/T216caad37819NpwJ368",
  "vcs_info": {
    "vcs_url": "//circleci.com/71555d36-fdfe-464d-95be-b0998480d9fb/b9b97877-5d65-4a25-9a77-5e841f086b56",
    "default_branch": "main",
    "provider": "CircleCI"
  }
}